app/services/breave_coin.js

```diff
@@ -0,0 +1,23 @@
+const request = require('request-promise');
+
+const { braveCoinEnpoint, braveHost, braveApiKey } = require('../../config/index').common.braveCoin;
```

Comment:
You can skip going to the index file, this is done by default. For example `const { ... } = require('../config')`